aksetup_helper.py

```diff
@@ -43,14 +43,21 @@ def setup(*args, **kwargs):
 
 
 def get_numpy_incpath():
-    from os.path import join, dirname
+    from os.path import join, dirname, exists
     from importlib.util import find_spec
     origin = find_spec("numpy").origin
     if origin is None:
         raise RuntimeError("origin of numpy package not found")
 
     pathname = dirname(origin)
```

Comment:
How about
```suggestion
    import numpy as np
    return np.get_include()
```
?

Reply:
I seem to recall we can't import numpy here, because it applies violence to distutils. This may no longer be relevant for 3.12 (which doesn't have that), but at least nominally, support for 3.8 is still on the table.

Reply:
There's this comment, which seems to back that up:

https://github.com/inducer/aksetup/commit/631165c4497f5f8cc711a8027b9fd6b04b3afe68#diff-47244c94d71d3c1c553c5947892cff5bc0b76988100273464ad175851f7a92d8R39

I most definitely don't remember exactly in what way distutils gets messed up.